src/gs-shell-overview.c

```diff
@@ -159,7 +159,6 @@ gs_shell_overview_get_popular_cb (GObject *source_object,
 		goto out;
 	}
 	/* Don't show apps from the category that's currently featured as the category of the day */
```

Comment:
If you simply add the "FALSE" below making it really easy to bring back that line, perhaps this should be in a similar state.  I would suggest either completely removing in both places, or do something like add a constant that is checked in both places.